course4/homework1/vector2d.h

```diff
@@ -2,8 +2,29 @@
 
 #include <ostream>
 
+
 struct Vector2d
 {
-	// TODO
+  int X;
```

Comment:
Bad naming. ALL CAPS names are usually used for constants, in most of the coding styles out there.